tools/assets/stylesheets/avaiga.css

```diff
@@ -1,25 +1,40 @@
+@import 'partials/_grid.css';
+
 :root {
   --md-primary-fg-color: #293183;
   --md-primary-fg-color--light: #f6921b;
   --md-primary-fg-color--dark: #ec693c;
+  --md-paper-bg-color: #fff;
+  --transition-duration-base: 400ms;
+  --tooltip-show-delay: 400ms;
+}
+[data-md-color-scheme="slate"] {
+  --md-hue: 240;
+  --md-paper-bg-color: #140a1e;
 }
+
 .md-header__button.md-logo img {
   width: 48px;
   height: 48px;
 }
 
-[data-md-color-scheme="slate"] {
-  --md-hue: 240;
+.md-content{
+  overflow: initial;
 }
 
-a.ext-link:after {
+/* Display the right icon for the right theme */
+.icon-light{ display: none; }
```

Comment:
Please make the CSS code homogeneous: block after selector on new line always or never.
Always would be pretty bad hu?

Reply:
There's a middle ground here though: "only for one CSS rule déclaration".
It keeps the readability when multiple one-rule selectors are written after one another.
But let's go with never, then, if you prefer. I adapt. 😉